litellm/model_prices_and_context_window_backup.json

```diff
@@ -2674,10 +2716,10 @@
         "max_tokens": 8192,
         "max_input_tokens": 2097152,
         "max_output_tokens": 8192,
-        "input_cost_per_token": 0.0000035,
-        "input_cost_per_token_above_128k_tokens": 0.000007,
-        "output_cost_per_token": 0.0000105,
-        "output_cost_per_token_above_128k_tokens": 0.000021,
+        "input_cost_per_token": 0,
```

Comment:
is gemini-pro free?

Reply:
isn't this the pricing? 
<img width="1146" alt="Screenshot 2024-09-13 at 9 12 19 PM" src="https://github.com/user-attachments/assets/6efeba51-cb44-49ae-be3e-9c4c8cb17a05">

Reply:
I did not change the pricing for Gemini Pro. That's a Git bug. If you expand the changes, you will see that I changed "gemini/gemini-1.5-pro-exp-0827".